crates/cdk/src/nuts/nut17/mod.rs

```diff
@@ -9,6 +13,20 @@ use crate::{
 };
 use serde::{Deserialize, Serialize};
 use std::ops::Deref;
+#[cfg(feature = "mint")]
+use std::sync::Arc;
```

Comment:
Can you reorder these imports to be consistent with the style guide. 

https://github.com/cashubtc/cdk/blob/main/CODE_STYLE.md#order-of-imports

Reply:
https://github.com/cashubtc/cdk/pull/449


fixed it here no need to hold up merging for that